packages/fast-components-react-msft/app/app.tsx

```diff
@@ -53,6 +55,10 @@ export default class App extends React.Component<{}, IAppState> {
             <Site
                 formChildOptions={formChildOptions}
                 onUpdateDirection={this.handleUpdateDirection}
+                onUpdateTheme={this.handleUpdateTheme}
+                /* tslint:disable:max-line-length */
+                themes={[{name: "light", background: DesignSystemDefaults.backgroundColor}, {name: "dark", background: DesignSystemDefaults.foregroundColor}]}
```

Comment:
can we clean this up by defining these as cost values outside the class? Then we don't ned to disable max line length